pkg/loader/fileloader.go

```diff
@@ -166,6 +166,10 @@ func (l *fileLoader) New(path string) (ifc.Loader, error) {
 		}
 		return newGitLoader(path, l.fSys, l.referrer, l.cloner)
 	}
+	if l.uri != "" {
```

Comment:
I think this restriction is too strong. I will also block the remote bases from accessing transitive bases in the same repo.

Reply:
transitive based in the same repo definitely seems useful

Reply:
yes, the way we track the "repo's dir" is a bit of a mess at the moment.
I can continue in this PR, or check this in, and loosen the restriction in a subsequent PR.

Reply:
meh, as long as nobody is panicking, i'll loosen it here.  will ping later.